script.js

```diff
@@ -0,0 +1,26 @@
+const webDevProjectsArray = [
+    {
+    Description: "Portfolio Site: This site was created using my newly developed skills in HTML, CSS, and Javascript. Hopefully this site will give you a great depiction of what I am capable of with just a few weeks of training under my belt.",
+}, 
+{
+    Description: "More to come!",
+}
+];
+const projectBtn = document.getElementById("projectButton")
+let currentIndex = 0
+
+
+function displayProject() {
+    if(currentIndex === webDevProjectsArray.length-1){
+        currentIndex = 0 
+    }else currentIndex++
+    const currentProjectEl = document.getElementById("currentProject")
```

Comment:
So if you add line 17 outside of the function (above it), then include this line also outside of your function:
`currentProjectEl.textContent = webDevProjectsArray[currentIndex].Description`
You should be able to remove the hard-coded HTML project. LMK if that doesn't make sense!